addons/event/migrations/9.0.0.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,127 @@
+---Fields in module 'event'---
+event        / event.event              / active (boolean)              : NEW
+# Nothing to do
+
+event        / event.event              / badge_front (html)            : NEW
+# Nothing to do
+
+event        / event.event              / color (integer)               : NEW
+# Nothing to do
+
+event        / event.event              / country_id (many2one)         : not a function anymore
+# Nothing to do as country_id is now a related field from address_id
+
+event        / event.event              / email_confirmation_id (many2one): DEL relation: email.template
+event        / event.event              / email_registration_id (many2one): DEL relation: email.template
+
+
+event        / event.event              / event_logo (html)             : NEW
+# Noting to do
+
+event        / event.event              / event_mail_ids (one2many)     : NEW relation: event.mail
+# Nothing to do
+
+event        / event.event              / message_follower_ids (many2many): not a function anymore
+event        / event.event              / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
+event        / event.event              / message_follower_ids (many2many): type is now 'one2many' ('many2many')
+# Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
+
+event        / event.event              / seats_availability (selection): NEW required: required, selection_keys: ['limited', 'unlimited'], req_default: unlimited
+# Updated from seats_max
+
+event        / event.event              / type (many2one)               : was renamed to event_type_id [nothing to to]
+# Nothing to do
+
+event        / event.mail               / event_id (many2one)           : NEW relation: event.event, required: required
```

Comment:
No, it isn't. The field contains the template of the mail that will be automatically sent. It isn't the same that email_confirmation_id and email_registration_id fields.